targets/targets.json

```diff
@@ -1465,10 +1465,11 @@
             }
         },
         "macros_add": ["RTC_LSI=1", "HSE_VALUE=12000000", "GNSSBAUD=9600"],
-        "device_has_add": ["ANALOGOUT", "TRNG"],
+        "device_has_add": ["ANALOGOUT", "SERIAL_FC", "TRNG", "FLASH"],
```

Comment:
@jeromecoutant's change to all the STM32F4 platforms, which I had to rebase against, [dropped it](https://github.com/ARMmbed/mbed-os/commit/4fc4405306bdea3c4f6435b79fcc96ff6c91374d#diff-3d6b492f3b697230670ddbb26104bbfdL1482), so I added it back in again.